teuthology/salt.py

```diff
@@ -53,9 +53,9 @@ def __init__(self, ctx, config, **kwargs):
         self._teuthology_ip_address = None
         self._minions = []
         if 'master_fqdn' not in kwargs:
-            self.master_fqdn(self.teuthology_fqdn)
+            self._master_fqdn(self.teuthology_fqdn)
         else:
-            self.master_fqdn(kwargs['master_fqdn'])
+            self._master_fqdn(kwargs['master_fqdn'])
 
```

Comment:
I thought I didn't need the leading underscore if I defined a setter called `master_fqdn` (see line 83)